### PR TITLE
Add information how to add a sequence as default value

### PR DESCRIPTION
**Pull request by andreasscherbaum**

### CODE REVIEW

**Comment by andreasscherbaum** [comment]
@mkiyama I don't know how to enumerate the new xml element. Shall I just add +1 to the last value, or do I need to store that information somewhere?

File: gpdb-doc/dita/admin_guide/ddl/ddl-sequence.xml [hunk 1]
```diff
@@ -76,4 +76,20 @@
       </p>
     </body>
   </topic>
+  <topic xml:lang="en">
+    <title>Making a Sequence the default value for a column</title>
```
Comment (mkiyama) [inline gpdb-doc/dita/admin_guide/ddl/ddl-sequence.xml]: Adding a 1 should be ok.
\<topic id="topic91" xml:lang="en">

Also the capitalization of the title should be
Making a Sequence the Default Value for a Column

Comment (andreasscherbaum) [inline gpdb-doc/dita/admin_guide/ddl/ddl-sequence.xml]: Merged

File: gpdb-doc/dita/admin_guide/ddl/ddl-sequence.xml [hunk 2]
```diff
@@ -76,4 +76,20 @@
...
+    <body>
+      <p>A sequence can be used in the <codeph>CREATE TABLE</codeph> command (instead of the
+          <codeph>SERIAL</codeph> or <codeph>BIGSERIAL</codeph> type):</p>
+      <p>
+        <codeblock>CREATE TABLE ... (
+            id INT4 DEFAULT nextval('id_seq')
+            );
+</codeblock>
+      </p>
+      <p>Alternatively a column can be altered and a sequence can be added as default value:</p>
```
Comment (mkiyama) [inline gpdb-doc/dita/admin_guide/ddl/ddl-sequence.xml]: Minor edit comma after Alternatively.
Alternatively,  a column

**Pull request merged**